packages/flutter_tools/test/commands.shard/permeable/create_test.dart

```diff
@@ -36,6 +37,7 @@ final Generator _kNoColorTerminalPlatform = () => FakePlatform.fromPlatform(cons
 final Map<Type, Generator> noColorTerminalOverride = <Type, Generator>{
   Platform: _kNoColorTerminalPlatform,
 };
+final BufferLogger logger = BufferLogger.test();
```

Comment:
Add this in `main()` under `LoggingProcessManager loggingProcessManager;

Reply:
done